armi/reactor/reactors.py

```diff
@@ -1825,10 +1825,10 @@ def findAllMeshPoints(self, assems=None, applySubMesh=True):
                     axisVal = float(base[axis])  # convert from numpy.float64
                     step = float(top[axis] - axisVal) / subdivisions
                     for _subdivision in range(subdivisions):
-                        collection.add(round(axisVal, units.FLOAT_DIMENSION_DECIMALS))
+                        collection.add(round(axisVal, 8))
                         axisVal += step
                     # add top too (only needed for last point)
-                    collection.add(round(axisVal, units.FLOAT_DIMENSION_DECIMALS))
+                    collection.add(round(axisVal, 8))
```

Comment:
I think a comment explaining the problem that was uncovered here could be useful. Or else we are liable to make the same mistake again sometime in the future.

Reply:
Addressed in https://github.com/terrapower/armi/pull/1225/commits/72139e657b7e1a02aa4a41e65b659ab47339f4c9